app/controllers/agencies_controller.rb

```diff
@@ -33,23 +33,26 @@ def create
     @back_url = session[:previous_url]
     @agency = Agency.new(agency_params.except(:jurisdiction))
     @agency.jurisdiction_type = agency_params[:jurisdiction]
-    if @agency.save
-      flash[:success] = 'Agency was successfully created.'
-      redirect_to @back_url, status: :created
-    else
-      render 'new'
+    @agency.jurisdiction = agency_params[:jurisdiction]
+    respond_to do |format|
```

Comment:
This should be changed to match the current format.

Reply:
Guess I did this or do you mean we get rid of `@agency.jurisdiction_type = agency_params[:jurisdiction]`

Reply:
Oh no, I mean that they use a different format than `respond_to` in master.  You should change it to match the format that's in master.